com.unity.netcode.gameobjects/Runtime/Core/NetworkObject.cs

```diff
@@ -1056,16 +1056,45 @@ internal NetworkBehaviour GetNetworkBehaviourAtOrderIndex(ushort index)
 
         internal struct SceneObject
         {
+            public ulong NetworkObjectId;
+            public ulong OwnerClientId;
             public struct HeaderData : INetworkSerializeByMemcpy
             {
-                public ulong NetworkObjectId;
-                public ulong OwnerClientId;
                 public uint Hash;
+                private byte m_BitField;
```

Comment:
Just leaving a reminder that this writes 8 bytes as opposed to 5 bytes.

Reply:
fixed